docs/pages/usage-billing.mdx

```diff
@@ -57,6 +57,25 @@ calculate two types of billing metrics:
 - Monthly Active Users
 - Teleport Protected Resources
 
+### Usage metrics in the WebUI
+
+  1. Go to https://teleport.sh/ and enter your tenant name.
```

Comment:
Should we mention that this will only show up if 
1) the customer is on a usage-based plan, and
2) the user has permission to read the billing resource